pages/clusters/overview.md

```diff
@@ -1,18 +1,40 @@
 # Clusters overview
 
-Clusters are a new way of managing your Buildkite agents. They allow teams to self-manage their agent pools, let admins create isolated sets of agents and pipelines within the one Buildkite organization, and help to make agents and queues more discoverable across your organization.
+>📘 Clusters
+> Clusters will be enabled for all organizations on February 26 2024.
+
+Clusters is a Buildkite feature used to manage and organize agents and queues, which:
+
+- allows teams to self-manage their Buildkite agent pools,
+- allows admins to create isolated sets of agents and pipelines within the one Buildkite organization,
+- helps make agents and queues more discoverable across your organization, and
+- provides easily accessible queue metrics.
 
 The following diagram shows the architecture with cluster enabled.
 
 <%= image "clusters-architecture.png", alt: "Diagram showing existing architecture and architecture with clusters" %>
 
 Clusters encapsulate groups of agents and pipelines, enabling the following:
 
-* Clusters are viewable to your entire organization, allowing engineers to better understand the agents and queues available for their pipelines.
-* Individual users or teams can maintain their own clusters. Cluster maintainers can manage queues and agent tokens and add and remove pipelines.
-* Pipelines can be assigned to a cluster, ensuring their builds run only on the agents connected to this cluster. These pipelines can also trigger builds only on other pipelines in the same cluster.
+- Clusters are viewable to your entire organization, allowing engineers to better understand the agents and queues available for their pipelines.
+- Individual users or teams can maintain their own clusters. Cluster maintainers can manage queues and agent tokens and add and remove pipelines.
+- Pipelines can be assigned to a cluster, ensuring their builds run only on the agents connected to this cluster. These pipelines can also trigger builds only on other pipelines in the same cluster.
+
+## Queue metrics
+
+Clusters provides additional, easy to access queue metrics that are available only for queues within a cluster.
+Learn more in [Cluster queue metrics](/docs/pipelines/cluster-queue-metrics).
+
+## Accessing clusters and agents
+
+The release of clusters will change navigating to your agents on the UI.
 
-## Enable clusters
+If you only have one Cluster with one Queue selecting _Agents_ in the global navigation will take you to your single queue.
```

Comment:
Is it worthwhile weaving into the narrative somewhere around here that when a new Buildkite org is created, the following is also created:

- 1 default cluster called **Default cluster**
- 1 default queue called **default-queue**
- 1 initial agent token called **Initial agent token**

?

Reply:
Consider revising with these suggestions post clusters GA because until then, this behaviour won't be observed.